SwiftUI_DZ/SwiftUI_DZ/InviteView/InviteViewModel.swift

```diff
@@ -5,10 +5,52 @@ import SwiftUI
 
 /// Модель экрана приглашения
 final class InviteViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let zeroValue = 0
+        static let emptyString = ""
+        static let loginTextFieldMaxLimitCharCount = 18
+        static let passwordMaxLimitInt = 15
+        static let passwordMinLimitInt = 6
+        static let zeroNumber = 0.0
+        static var progressMaxCount = 4.0
+        static var timeInterval = 0.5
+        static var oneNumber = 1
+        static let fiveDouble = 5.0
+    }
+
     // MARK: - Public Properties
 
+    @Published var isStartAnimation = false
     @Published var isTabViewShown = false
     @Published var isVerificateViewShown = false
     @Published var isAutorizeShown = false
+    @Published var isUserView = false
```

Comment:
```suggestion
    @Published var isUserView = false
```
чет название не очень